ietf-gss-kex-qr.xml

```diff
@@ -133,6 +135,198 @@
           <c>gss-qr-sha256-</c><c>SHA-256</c>
           <c>gss-qr-sha512-</c><c>SHA-512</c>
       </texttable>
+      <section title="Generic Quantum Resistant GSS-API key Exchange">
+        <t>This section reuses much of the scheme defined in Section 2.1
+            of <xref target="RFC4462"/> though it does not transport FFDH
+            key shares in the exchanged messages.
+        </t>
+        <t>This section defers to <xref target="RFC7546"/> as the source of
+            information on GSS-API context establishment operations,
+            Section 3 being the most relevant. All security considerations
+            described in <xref target="RFC7546"/> apply here too.
+        </t>
+        <t>The parties generate in the key exchange generate nonces.
```

Comment:
could there be an extra generate in this sentence

Reply:
fixed